packages/http-client-java/generator/http-client-generator-core/src/main/java/com/microsoft/typespec/http/client/generator/core/util/CodeNamer.java

```diff
@@ -94,6 +94,15 @@ public static String getEnumMemberName(String name) {
             return name;
         }
 
+        // Special handling for inputs that start with special characters followed by numbers
+        if (name.length() > 1 && !Character.isLetterOrDigit(name.charAt(0)) && Character.isDigit(name.charAt(1))) {
```

Comment:
I could add `name.length() >= 2` if that is more readable/less of an index out of bounds mind phrack